packages/@okta/vuepress-site/docs/reference/api/sessions/index.md

```diff
@@ -675,49 +639,49 @@ HTTP/1.1 204 No Content
+>**Note:** The `mfaActive` parameter is a <ApiLifecycle access="deprecated" /> feature. Use the `lastFactorVerification` attribute in conjunction with `amr` to understand if the user has performed MFA for the current Session. Use the [Factors API](/docs/reference/api/factors/#list-enrolled-factors) to query the factor enrollment status for a given user.
 
-#### Optional Session Properties
+#### Optional Session properties
 
 The [Create Session](#create-session-with-session-token) operation can optionally return the following properties when requested.
 
 | Property                                      | Description                                                                                                                                                                       |
 | --------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| cookieToken                                   | Another one-time token which can be used to obtain a session cookie by visiting either an application's embed link or a session redirect URL.                                     |
-| cookieTokenUrl                                | URL for a a transparent 1x1 pixel image which contains a one-time session token which when visited sets the session cookie in your browser for your organization.                 |
+| cookieToken                                   | Another one-time token that is used to obtain a session cookie by visiting either an application's embed link or a Session redirect URL.                                     |
+| cookieTokenUrl                                | The URL for a transparent 1x1 pixel image that contains a one-time session token that, when visited, sets the session cookie in your browser for your organization.                 |
 
-> The `cookieToken` is a <ApiLifecycle access="deprecated" /> property. Instead, use the [Authentication API](/docs/reference/api/authn/), which supports the full user authentication pipeline and produces a `sessionToken` which can be used in this API.
+>**Note:** The `cookieToken` is a <ApiLifecycle access="deprecated" /> property. Instead, use the [Authentication API](/docs/reference/api/authn/) that supports the full user authentication pipeline and produces a `sessionToken` that you can use in this API.
 
-> The `cookieTokenUrl` is a <ApiLifecycle access="deprecated" /> property, because modern browsers block cookies set via embedding images from another origin (cross-domain).
+>**Note:** The `cookieTokenUrl` is a <ApiLifecycle access="deprecated" /> property, because modern browsers block cookies that are set through embedding images from another origin (cross-domain).
```

Comment:
with this one, I was wrong. Let's just leave this as content. I can show you later today how to do a note with bullets inside of the note frame. Remind me.